src/experimental/tools/mediaCapabilitiesProber/probers/__tests__/decodingInfo.test.ts

```diff
@@ -82,22 +80,21 @@ describe("MediaCapabilitiesProber probers - decodingInfo", () => {
       },
     };
 
-    expect.assertions(2);
```

Comment:
I just learned about `expect.assertions()` is it something that should be avoided?

Reply:
I don't like using it because it's something easy to forget about and it's not really a meaningful test beside ensuring that a code path with assertions has been taken, in which case I prefer a more explicit check like a boolean.